src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/memory/FileMapInfo.java

```diff
@@ -168,23 +169,58 @@ public void createVtableTypeMapping() {
       vTableTypeMap = new HashMap<Address, Type>();
       long addressSize = VM.getVM().getAddressSize();
 
-      // vtablesIndex points to this:
-      //     class CppVtableInfo {
-      //         intptr_t _vtable_size;
-      //         intptr_t _cloned_vtable[1];
-      //         ...
-      //     };
-      //     CppVtableInfo** CppVtables::_index;
-      // This is the index of all the cloned vtables. E.g., for
+      // vtablesIndex points to to an array like this:
+      // long info[] = {
+      //   offset of the CppVtableInfo for ConstantPool,
+      //   offset of the CppVtableInfo for InstanceKlass,
+      //   offset of the CppVtableInfo for InstanceClassLoaderKlass,
+      //   ...
+      // };
+      //
+      // class CppVtableInfo {
+      //   intptr_t _vtable_size;
+      //   intptr_t _cloned_vtable[1];
+      //   ...
+      // };
+      //
+      // The following loop compues the following
```

Comment:
Since there are lot of comments after this point, maybe the wording should instead be "The loop below...". Also, should be "computes"